sources/packages/api/src/route-controllers/application/application.student.controller.ts

```diff
@@ -131,8 +183,18 @@ export class ApplicationController extends BaseController {
    */
   @CheckSinValidation()
   @CheckRestrictions()
-  @AllowAuthorizedParty(AuthorizedParties.student)
   @Patch(":applicationId/submit")
+  @ApiOkResponse({ description: "Application submitted" })
+  @ApiUnprocessableEntityResponse({
+    description:
+      "Program Year is not active. OR  Invalid study dates. OR \
```

Comment:
Can we convert this for more usual documentation? I understood the point of using the "OR" but I would still recommend using a more natural language.

Reply:
what if a single description itself has an `OR`, it will confuse the reader right?